ui/app/AppLayouts/WalletV2/components/AddAccountPopup.qml

```diff
@@ -0,0 +1,227 @@
+                    color: Style.current.textColor
+                }
+                Rectangle {
+                    width: parent.width
+                    height: 56
+                    anchors.top: inputLabel.bottom
+                    anchors.topMargin: 7
+                    radius: 10
+                    color: "pink"
+                    opacity: 0.6
+                }
+            }
+        }
+
+        Input {
+            id: passwordInput
+            anchors.left: parent.left
+            anchors.leftMargin: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 10
+            implicitHeight: 82
+            customHeight: 56
+            anchors.top: accountNameInputRow.bottom
+            anchors.topMargin: popup.marginBetweenInputs
+            placeholderText: qsTr("Give the account a secure password")
+            label: qsTr("Password")
+            textField.echoMode: TextInput.Password
+            validationError: popup.passwordValidationError
+        }
+    }
+    footer: StatusButton {
+        anchors.top: parent.top
+        anchors.right: parent.right
+        text: popup.loading ? qsTrId("loading") : qsTrId("add-account")
+        enabled: !popup.loading && (passwordInput.text !== "") && (accountNameInput.text !== "")
+                 && (keyOrSeedPhraseInput.correctWordCount || (keyOrSeedPhraseInput.text !== ""))
+
+        MessageDialog {
+            id: accountError
+            title: qsTr("Adding the account failed")
+            icon: StandardIcon.Critical
+            standardButtons: StandardButton.Ok
+        }
+
+        onClicked : {
+            loading = true;
+            if (!popup.validate() || (popup.seedPhraseInserted && !popup.validateSeed())) {
+                errorSound.play();
+                popup.loading = false;
+            } else {
+                //TODO account color to be verified with design
+                const result = seedPhraseInserted ?
+                                 walletModel.accountsView.addAccountsFromSeed(keyOrSeedPhraseInput.text, passwordInput.text, accountNameInput.text, "") :
+                                 walletModel.accountsView.addAccountsFromPrivateKey(keyOrSeedPhraseInput.text, passwordInput.text, accountNameInput.text, "");
+                popup.loading = false;
+                if (result) {
+                    let resultJson = JSON.parse(result);
+                    errorSound.play();
+                    if (Utils.isInvalidPasswordMessage(resultJson.error)) {
+                        popup.passwordValidationError = qsTrId("wrong-password");
+                    } else {
+                        accountError.text = resultJson.error;
+                        accountError.open();
+                    }
+                    return;
+                }
+                popup.onAfterAddAccount();
+                popup.reset();
+                popup.close();
+            }
+        }
+    }
+}
```

Comment:
I might be wrong here, but I believe all occurances of `qsTrId()` need to be replaced with `qsTr()` because the stored translations don't know this source file yet, hence i18n ids won't match the location of this file.